### ISSUE TITLE
Setup Instructions Outdated

### ISSUE THREAD

**Issue by pmarcellino** [jupyterlab/jupyterlab-google-drive#178]
I was trying to use this plugin and went through the [tutorial](https://github.com/jupyterlab/jupyterlab-google-drive/blob/master/docs/setup.md) but many of the steps have been changed by google, resulting in quite different looking screens and options. I ended up being un succesfull in being able to get this to work.
![image](https://user-images.githubusercontent.com/46568443/84929105-fcbb4080-b094-11ea-8e36-b024d0ed14a7.png)

**Comment by bmshouse** [comment]
@pmarcellino At some point, Google started to disallow localhost (or 127.0.0.1). Would you try using *127.0.0.1.xip.io* anywhere you see *localhost* and let me know if that gets it working? If it does I will start work on re-writing the directions with this alternative. Thank you.

**Comment by scottrk** [comment]
Ran across this issue which is also plaguing me. I don't know if the suggested fix worked for the individual that opened the issue, but I tried with both 127.0.0.1 and 127.0.0.1.xip.io without success. The exact same error message was given in both cases.

**Comment by bmshouse** [comment]
@scottrk @pmarcellino I am going over the setup document and the error message. It looks like it is complaining because you are trying to access you local Jupyter using `localhost` instead of `127.0.0.1.xip.io`

Would you please try to use the allowed domain name when accessing Jupyter and let me know if it makes any difference?

Thank you.

**Comment by scottrk** [comment]
I'm still getting this issue. I even set up a local certificate to allow for access via https and tried a variety of different ports all with the same result with the exception that the url changed.

**Comment by jadm333** [comment]
So I was having the same problem and after tried different things this was the way it work and I'm not sure wich one it's the actual fix:

1. OAuth consent config:
Same as [setup](https://github.com/jupyterlab/jupyterlab-google-drive/blob/master/docs/setup.md)

2. OAuth client ID config:
**Authorized javascript origins:**
https://127.0.0.1.xip.io
http://localhost:8888
http://127.0.0.1:8888
**Authorized redirect URI's:**
https://127.0.0.1.xip.io/oauth_callback
http://localhost:8888/lab
http://localhost:8888/lab
http://127.0.0.1:8888/oauth_callback
http://127.0.0.1:8888/lab
http://localhost

3. And most important it only works on Firefox web browser and in the URL: http://localhost:8888/lab. If I try it on chrome it pop up the same **Not a valid origin for the clie...** 

Hope it helps

**Comment by hovikgas** [comment]
Yeah, it seems like they want https... is there a way to run Jupyter Lab/Notebook with https, instead of http? 

I keep getting the following error: 

Error 400: invalid_request
Permission denied to generate login hint for target domain.
